synchronizer/synchronizer.go

```diff
@@ -562,12 +563,18 @@ func (s *ClientSynchronizer) RequestAndProcessRollupGenesisBlock(dbTx pgx.Tx, la
 		log.Error("error getting rollupInfoByBlockRange after set the genesis: ", err)
 		return err
 	}
-	// Check that the response is the expected. It should be 1 block with 2 orders
+	// Check that the response is the expected. It should be 1 block with ForkID event
+	log.Debugf("genesis block (%d) events: %+v", lastEthBlockSynced.BlockNumber, order)
 	err = sanityCheckForGenesisBlockRollupInfo(blocks, order)
 	if err != nil {
 		return err
 	}
-	forkId := s.state.GetForkIDByBlockNumber(blocks[0].BlockNumber)
+	log.Infof("Processing genesis block %d orders: %+v", lastEthBlockSynced.BlockNumber, order)
+	err = s.internalProcessBlock(blocks[0], order[blocks[0].BlockHash], false, dbTx)
+	if err != nil {
+		log.Error("error processinge events on genesis block %d:  err:%w", lastEthBlockSynced.BlockNumber, err)
```

Comment:
Should we use `Errorf` here perhaps?

```suggestion
		log.Errorf("error processinge events on genesis block %d: err:%w", lastEthBlockSynced.BlockNumber, err)
```